docs/reference/contributing/connectivity/CellularInterface.md

```diff
@@ -116,40 +87,52 @@ Setup for some predefined targets is available in [CellularTargets.h](https://os
+
+##### Providing an implementation using the Mbed OS provided network stacks (PPP mode)
+
+If the new modem supports PPP mode, then you can use the existing Mbed OS LWIP stack to control the modem when connecting to a network. To use the LWIP stack, set `lwip.ppp-enabled` to true in an application's `mbed_app.json`. [UBLOX PPP](https://github.com/ARMmbed/mbed-os/tree/master/features/cellular/framework/targets/UBLOX/PPP) is an example of a modem implementation using PPP mode to connect to a network. For example, [mbed-os-example-cellular](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-cellular/) instantiates [EasyCellularConnection class](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_easy_cellular_connection_8h_source.html), which in turn instantiates [CellularConnectionFSM class](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_cellular_connection_f_s_m_8h_source.html). CellularConnectionFSM instantiates classes implementing [the AT command layer](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_a_t___cellular_device_8h_source.html) between the modem and the Mbed OS CPU. For PPP classes [UBLOX_PPP](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP.h), [UBLOX_PPP_CellularNetwork](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP_CellularNetwork.h) and [UBLOX_PPP_CellularPower](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP_CellularPower.h) have been implemented for a modem specific functionality. In the AT command layer, [AT_CellularNetwork](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_a_t___cellular_network_8h_source.html) includes functionality calling [nsapi_ppp_connect()](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/netsocket_2nsapi__ppp_8h_source.html) to start the data call through the PPP pipe.
```

Comment:
Similar comment to above.
The information here is very packed and hard to read.
Can we split and restructure this?